bids/layout/layout.py

```diff
@@ -521,6 +539,15 @@ def files(self):
         """Get the files."""
         return self.get_files()
 
+    @classmethod
+    def load_from_db(cls, database_path):
```

Comment:
I would suggest keeping consistent with `save()`.

```suggestion
    def load(cls, database_path):
```

Reply:
agree but i swore you or @tyarkoni suggested `load_from_db`